server/triffidsapi/api.py

```diff
@@ -1,54 +1,54 @@
-#!flask/bin/python
-from flask import Flask, jsonify, abort, make_response, request
-from flask_cors import CORS
+"""
+api.py
+- api endpoints for the Triffids server
+"""
 
-import sys
-sys.path.append('triffidsapi')
+from flask import Blueprint, make_response, abort, jsonify, request
 
-import parks
-import trees
-import benches
+from . import trees, parks, benches
+# import triffidsapi.trees as trees
+# import triffidsapi.parks as parks
+# import triffidsapi.benches as benches
 
-app = Flask(__name__)
-CORS(app)
+api = Blueprint('api', __name__)
 
 
-@app.errorhandler(404)
+@api.errorhandler(404)
 def not_found(error):
     return make_response(jsonify({'error': 'Not found'}), 404)
 
 
-@app.route('/')
+@api.route('/')
```

Comment:
Is this required or should we just remove it / replace it with something?

Reply:
Without the decorators, the routes will not work, AFAIK.